### ISSUE TITLE
Overall: More color, less gray

### ISSUE THREAD

**Issue by yatil** [w3c/wai-tutorials#129]
**Original Comment:**

> The colours - or lack of need attention. It would be great if there were some nice touches of colour to brighten it up. Too much grey!

**Comment by yatil** [comment]
See #182.
